src/NzbDrone.Core.Test/ParserTests/UnicodeReleaseParserFixture.cs

```diff
@@ -79,6 +79,8 @@ public void should_parse_chinese_multiepisode_releases(string postTitle, string
 
         [TestCase("[GM-Team][国漫][斗罗大陆][Anime Title][Douro Mainland][2019][215][AVC][GB][1080P]", "Anime Title", 215)]
         [TestCase("[GM-Team][国漫][Anime Title][Douro Mainland][2019][234][AVC][GB][1080P]", "Anime Title", 234)]
+        [TestCase("[GM-Team][国漫][Anime Title][2019][234][AVC][GB][1080P]", "Anime Title", 234)]
+        [TestCase("[GM-Team][国漫][Anime Title][2019][234 END][AVC][GB][1080P]", "Anime Title", 234)]
```

Comment:
This test case isn't passing currently: https://build.sonarr.tv/buildConfiguration/Sonarr_V4PullRequests_StatusReport/181141?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildTestsSection=true

Reply:
![image](https://user-images.githubusercontent.com/49138427/205598250-2aa67fba-4ce0-41cd-bd38-126cd63764a1.png)
I use a online regex site to test my case, https://regex101.com/

Reply:
You can login as guest.

```
Expected value to be 234, but found 2019.
   at FluentAssertions.Execution.LateBoundTestFramework.Throw(String message)
   at FluentAssertions.Execution.TestFrameworkProvider.Throw(String message)
   at FluentAssertions.Execution.DefaultAssertionStrategy.HandleFailure(String message)
   at FluentAssertions.Execution.AssertionScope.FailWith(Func`1 failReasonFunc)
   at FluentAssertions.Numeric.NumericAssertions`1.Be(T expected, String because, Object[] becauseArgs)
   at NzbDrone.Core.Test.ParserTests.UnicodeReleaseParserFixture.should_parse_gm_team_releases_and_files(String postTitle, String title, Int32 absoluteEpisodeNumber) in M:\BuildAgent\work\853acb111afc90d8\src\NzbDrone.Core.Test\ParserTests\UnicodeReleaseParserFixture.cs:line 88
```